mws/apimws/migrations/0008_phplibs.py

```diff
@@ -4,6 +4,96 @@
 from django.db import models, migrations
 
 
+php_lib_values = [
```

Comment:
Shouldn't this be a new migration? Otherwise, deployments where migration 0008 has already run won't get this?

Maybe move to 0011?